cmd/chihaya/config.go

```diff
@@ -94,6 +95,19 @@ func (cfg ConfigFile) CreateHooks() (preHooks, postHooks []middleware.Hook, err
 				return nil, nil, errors.New("invalid client approval middleware config: " + err.Error())
 			}
 			preHooks = append(preHooks, hook)
+		case "deniability":
+			var hCfg deniability.Config
+			err := yaml.Unmarshal(cfgBytes, &hCfg)
+			if err != nil {
+				return nil, nil, errors.New("invalid deniability middleware config: " + err.Error())
+			}
+			hook, err := deniability.New(hCfg)
+			if err != nil {
+				return nil, nil, errors.New("invalid deniability middleware config: " + err.Error())
+			}
+
+			// TODO this needs to run after the responseHook
```

Comment:
Is this going to be an issue? Do we need to make the default response another middleware that you can order in your config?